Dockerfile

```diff
@@ -27,14 +26,27 @@ RUN ~/.local/bin/meson --buildtype=release build && cd build \
 	&& ninja \
 	&& ninja install
 
+FROM registry.access.redhat.com/ubi9/ubi-micro
+
 ENV LD_LIBRARY_PATH=$$LD_LIBRARY_PATH:/usr/local/lib:/usr/local/lib64
+COPY --from=builder /etc/login.defs /etc/login.defs
+COPY --from=builder /etc/security/pwquality.conf /etc/security/pwquality.conf
+COPY --from=builder /usr/local/bin/g729-codec-service /usr/local/bin/g729-codec-service
+COPY --from=builder /usr/local/lib64/libbcg729.so /usr/local/lib64
+COPY --from=builder /usr/local/lib64/libusockets.so /usr/local/lib64
+COPY --from=builder /usr/local/lib64/libz.so /usr/local/lib64
```

Comment:
Only suggestion is for some consistency on the ways files are being copied, i.e copied to a directory or to a specific file name:
```
COPY --from=builder /usr/local/lib64/libz.so /usr/local/lib64

```

vs 

```
COPY --from=builder /usr/lib64/libstdc++.so.6 /usr/lib64/libstdc++.so.6
```